packages/lumx-icons/rollup.config.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import glob from 'glob';
+import resolve from '@rollup/plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+import copy from 'rollup-plugin-copy';
+import cleaner from 'rollup-plugin-cleaner';
+
+import pkg from './package.json';
+
+const CONFIGS = require('../../configs');
+
+const DIST_PATH = path.resolve(__dirname, pkg.publishConfig.directory);
+export const extensions = ['.js'];
+
+export default {
+    input: 'index.js',
+    output: {
+        format: 'esm',
+        sourcemap: true,
+        dir: DIST_PATH,
+    },
+    plugins: [
+        /** Clean dist dir */
+        cleaner({ targets: [DIST_PATH] }),
+        /** Resolve source files. */
+        resolve({ browser: true, extensions }),
+        /** Transpile JS. */
+        babel({
+            extensions,
+            exclude: /node_modules/,
+            plugins: CONFIGS.babel.plugins,
+            presets: [
+                ['@babel/preset-env', { targets: 'defaults' }],
+            ],
+        }),
+        /** Copy additional files to dist. */
+        copy({
+            targets: glob.sync('*')
```

Comment:
Not sure if we should put all files in the `dist` folder? Filtering the "source" files could make our library leaner. (It wouldn't make much difference in this case, I agree :p)

Reply:
I could list `*.scss`, `*.js`, `*.d.ts`, `package.json`, `*.md` (README/LICENCE). The result would be the same because we need to bundle all files but at least it would be more explicit than `*`